Code/Framework/AzCore/Tests/StreamerTests.cpp

```diff
@@ -251,6 +252,9 @@ namespace AZ::IO
             AZ::AllocatorInstance<AZ::PoolAllocator>::Create();
             AZ::AllocatorInstance<AZ::ThreadPoolAllocator>::Create();
 
+            m_taskExecutor = AZStd::make_unique<TaskExecutor>();
+            TaskExecutor::SetInstance(m_taskExecutor.get());
```

Comment:
Ugh, so the TaskExecutor is using the FileIOBase pattern and not the AZ::`Interface` pattern?